content/en/tracing/trace_collection/otel_instrumentation/java.md

```diff
@@ -19,22 +19,62 @@ further_reading:
 
 The following OpenTelemetry features implemented in the Datadog library as noted:
 
```

Comment:
I'd suggest noting that for those listed as "Unsupported", there may be progress on moving those to "Supported" in the near future, and if a customer wants one of those features, they should escalate to support about this.

Reply:
Agreed. I just add the instrumentation annotation for example.
But the contributing guideline does not recommend such content ([ref](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#tense)):

> Avoid temporal words like "currently", "now", "will", etc. Describe the present state of the product.

So, what would be the right sentence to add below this table?

> The support of OpenTelemetry features is an ongoing effort. If there is any missing feature of your interest, please escalate to support about it.